AISKU/snippet/snippet.js

```diff
@@ -11,7 +11,7 @@ var aisdk = window[aiName] || (function (aiConfig) {
     var localDocument = document, localWindow = window, scriptText = "script";
     setTimeout(function () {
         var scriptElement = localDocument.createElement(scriptText);
-        scriptElement.src = aiConfig.url || "https://az416426.vo.msecnd.net/beta/ai.1.min.js";
+        scriptElement.src = aiConfig.url || "https://az416426.vo.msecnd.net/next/ai.1.min.js";
```

Comment:
What is this purpose of next?

Reply:
Using a different container name going forward. Old container was called `beta`, using `next` now

Reply:
So next will always have latest, is that correct?

Reply:
I think we will be doing it like this
`scripts/b`: latest stable version
`next`: next stable version (for flighting)
`beta`: additional experimental flighting endpoint